boards/arm/pan1781_evb/pan1781_evb_defconfig

```diff
@@ -0,0 +1,36 @@
+# PAN1781 EVB configuration
+
+# Copyright (c) 2022 Panasonic Industrial Devices Europe GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52820_QDAA=y
+CONFIG_BOARD_PAN1781_EVB=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# additional board options
+CONFIG_GPIO_AS_PINRESET=y
+
+# bluetooth advertising name
+CONFIG_BT_DEVICE_NAME="pan1781_evb"
```

Comment:
this should not be at board level, applications need to choose their name.

Reply:
Yes i have already seen the warning, when building an application, with this configuration. The reason, why i have added  this configuration to the *_defconfig file is that all of our boards should initially run with the same software. When i add the advertising name to the application, all of our boards would advertise with the same name. So our solution was to add this board specific naming to the files that define the boards, so the *_defconfig configuration. Hope that way of configuration is okay for you.

Reply:
The board defconfigs need to be kept minimal per porting guidelines, this option is clearly not minimal (depends on BT). You can add it to your application `prj.conf`, see for example https://github.com/teslabs/zds-2022-drivers-app/blob/main/app/prj.conf#L19

Reply:
Yes, you are right. I can also handle this on application level, by asking for the CONFIG_BOARD_PAN1780_EVB. I will remove the CONFIG_BT_DEVICE_NAME from *_defconfig files.